### PR TITLE
Adding recipients json file

### LINKED ISSUE #7: Get list of officials/representatives to send emails too

### PR DESCRIPTION
**Pull request by doshmajhan**
~~So `messages.py` is what the other repo was using to generate its templates. It's based off of the same templates we were looking at but broke the templates up into their individual pieces, so that they can be combined differently, giving a _semi_-unique template. I think it's a good approach that we could improve on so that it gives people a feeling that their message will be different from others.~~ 

Also `recipeints.json` is a reformatted list from the other repo of the council members that it was sending emails to. We can fill this list out more and maybe even organize it further based on local council members vs federal level ones.

~~Also `messages.py` is in python, but overall I was leaning towards just doing this whole project in React, I think it makes sense as we can accomplish everything in React that we could in Python, and will be easier to host/deploy. I'm low-key trash at React but I can scrape my way along. Let me know y'all thoughts.~~

### CODE REVIEW

**Comment by doshmajhan** [comment]
Damn, we can only have one reviewer that's wack. Guess I'll just tag everyone @philipbed @qxh5696 @RiceOnly @mellamoBrandon @danielroach9 @tqiu8 @bellyflopp 

If we make the repo public we may be able to get multiple reviewers.

**Comment by doshmajhan** [comment]
> Damn, we can only have one reviewer that's wack. Guess I'll just tag everyone @philipbed @qxh5696 @RiceOnly @mellamoBrandon @danielroach9 @tqiu8 @bellyflopp
> 
> If we make the repo public we may be able to get multiple reviewers.

It's public now

**Review by RiceOnly — COMMENTED** [review]
I like the customization

File: messages.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+# Generates the greeting to the recipient of the email
+def gen_greeting(person):
+    s = ["Dear", "Hello", "Greetings", "Hi"]
+    return "%s %s,\n\n" % (random.choice(s), person)
+
+# Prepends greeting statement to a user-generated message
+def attach_greeting(person, body):
+    s = ["Dear", "Hello", "Greetings", "Hi"]
+    return "%s %s,\n\n%s" % (random.choice(s), person, body)
```
Comment (bellyflopp) [inline messages.py]: What's the point of attach_greeting, if we can either call gen_greeting independently/are already call gen_greeting in the gen_body method?

Comment (doshmajhan) [reply messages.py]: No real point was prob just for ease of use. This is just code I took from the other repo, I haven't made any modifications, was more putting this up as an example for what we could do.

**Review by philipbed — COMMENTED** [review]
I think this is a good start. The random choices don't have a huge benefit.

Were you planning on editing this before generating the link? Or were you trying to use the content here to start this?

**Comment by doshmajhan** [comment]
> I think this is a good start. The random choices don't have a huge benefit.
> 
> Were you planning on editing this before generating the link? Or were you trying to use the content here to start this?

Was just using this as starter content, can edit it as we go forward.

**Comment by doshmajhan** [comment]
Updated to remove `messages.py`, we'll create our own template, this PR will just be adding the beginning of the recipients and their info.

**Review by philipbed — APPROVED** [review]
🚢 that thang

**Pull request merged**